homeassistant/components/calendar/__init__.py

```diff
@@ -132,12 +139,19 @@ def state_attributes(self):
 
         event = normalize_event(event)
         return {
+            "event_id": event["event_id"],
```

Comment:
@balloob Do we want to extend the common calendar event like this?

Reply:
That allows the user to see the event id in the state attributes so that they can either update the event or delete it